db/mainphotos.sql

```diff
@@ -0,0 +1,80 @@
+DROP DATABASE IF EXISTS mainphotos;
+CREATE DATABASE mainphotos;
+
+\c mainphotos
+
+CREATE SCHEMA IF NOT EXISTS mainphotos;
+SET search_path TO mainphotos, public;
+
+CREATE TABLE mainphotos.neighborhoods (
```

Comment:
Your naming convention works but I think with pqsl you can just call the table: 
- neighborhoods 
instead of mainphotos.neighborhoods
Same with the rest of the tables

Reply:
I wanted to ensure that it was created under the mainphotos schema, but with further testing it seems that setting the search path in line 7 would result in the "schema"."table_name" being redundant. Thanks!